Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/resource_printer.go

```diff
@@ -707,6 +727,11 @@ func printReplicationController(controller *api.ReplicationController, w io.Writ
 	name := controller.Name
 	namespace := controller.Namespace
 	containers := controller.Spec.Template.Spec.Containers
+	kind := options.KindName
```

Comment:
This adds to my comment https://github.com/openshift/origin/pull/9334/files#r69782642.

We are printing a Replication Controller. The kind is "replicationcontroller" or "rc". The "kind name" shouldn't be something that comes from the printer options... There's a mismatch in terms of who should know what.